### ISSUE TITLE
Unable to install from Community Plugins inside Obsidian

### ISSUE THREAD

**Issue by Exomancer** [Snezhig/obsidian-front-matter-title#14]
Congratulations on getting your plugin accepted into the Obsidian community plugins.  I cannot install it from the Community Plugins inside Obsidian.  I get an error "no appropriate found".

**Comment by Snezhig** [comment]
Thank you.
Have you tried to install this plugin on mobile or desktop?
You can install this plugin if you obsidian version is 0.14.6 at least.
As I know mobile version is 0.14.2.
Also I think that this plugin works with 0.14.2 correctly and I want to test it to reduce version requirement.
Until I do it you can use [BRAT](https://github.com/TfTHacker/obsidian42-brat) plugin to install.

**Comment by Exomancer** [comment]
I tried to install it on a mobile v1.2.0 (50) API v0.14.2 on Android.  I have been manually installing it from Github by downloading the .ZIP file which works just fine.  I just can't load it from inside the Community Plugins section inside Obsidian.  It gives me a "no appropriate version" error.  I just tried it again.  Same error message.  It's like it can't find the download to install it or there's a missmatch between the version requested and the version found.
